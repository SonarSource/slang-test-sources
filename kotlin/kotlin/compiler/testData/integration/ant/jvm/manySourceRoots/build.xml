<project name="Ant Task Test" default="build">
    <taskdef resource="org/jetbrains/kotlin/ant/antlib.xml" classpath="${kotlin.lib}/kotlin-ant.jar"/>

    <target name="build">
        <kotlinc output="${temp}/hello.jar">
          <src path="${test.data}/root1"/>
          <src path="${test.data}/root2"/>
        </kotlinc>
    </target>
</project>
