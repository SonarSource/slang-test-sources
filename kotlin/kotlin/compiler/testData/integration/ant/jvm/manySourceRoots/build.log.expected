OUT:
Buildfile: [TestData]/build.xml

build:
  [kotlinc] Compiling [[TestData]/root1, [TestData]/root2] => [[Temp]/hello.jar]

BUILD SUCCESSFUL
Total time: [time]

Return code: 0
